scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-3x.yaml

```diff
@@ -358,6 +358,69 @@ QFN-32-1EP_5x5mm_P0.5mm_EP3.65x3.65mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-32-1EP_5x5mm_P0.5mm_EP3.7x3.7mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.espressif.com/sites/default/files/documentation/0a-esp8285_datasheet_en.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    minimum: 4.95
+    nominal: 5.00
+    maximum: 5.05
+  body_size_y:
+    minimum: 4.95
+    nominal: 5.00
+    maximum: 5.05
+  body_height:
```

Comment:
I noticed that body height is not quite the right parameter name. Could you change it to `overall_height`?

Do not change this in any other part. I do this already as part of one of my pull requests so that would only create more merge conflicts.